src/assets/stylesheets/Instructions.scss

```diff
@@ -18,6 +18,20 @@
     max-width: 100%;
   }
 
+  table {
+    border-collapse: collapse;
+    margin-bottom: $space-1-5;
+    inline-size: 100%;
+
+    tr {
+      border-bottom: 1px solid $rpf-black;
```

Comment:
Replace with: `border-block-end: 1px solid $rpf-black`;

Reply:
Ah yes, thanks for spotting this 👍